### ISSUE TITLE
The SDK doesn't set mfxExtVideoSignalInfo::VideoFullRange for jpeg decoding.

### ISSUE THREAD

**Issue by xhaihao** [Intel-Media-SDK/MediaSDK#2915]
call `MFXVideoDECODE_DecodeHeader` with an attached `MFX_EXTBUFF_VIDEO_SIGNAL_INFO` buffer for jpeg decoding, however mfxExtVideoSignalInfo::VideoFullRange is not set for a full-range jpeg file.

**Comment by wangyan-intel** [comment]
@chuanli1 Could you please take a look? If confirm it, we can assign one guy to fix it. Thanks

**Comment by chuanli1** [comment]
> @chuanli1 Could you please take a look? If confirm it, we can assign one guy to fix it. Thanks
Sure, I will check it.

**Comment by chuanli1** [comment]
Hi @xhaihao, could you provide related clip and an app patch of attached MFX_EXTBUFF_VIDEO_SIGNAL_INFO buffer? Thanks.

**Comment by xhaihao** [comment]
@chuanli1 you may use a yuvj420p jpeg file 
```
$ ffprobe in.jpeg 2>&1 | grep yuvj420p 

$ ffmpeg -hwaccel qsv -c:v mjpeg_qsv -i in.jpeg -f null -
```

**Comment by uartie** [comment]
@chuanli1 @dmitryermilov any update on this issue?

**Comment by chuanli1** [comment]
I check jpeg spec(itu-t81) and don’t find this color range syntax, so RT don't parser this flag from Jpeg header, and HW don’t need this flag for decoder, and this issue is not a bug, but a new feature.

**Comment by xhaihao** [comment]
@chuanli1  If so , may you set mfxExtVideoSignalInfo::VideoFullRange to 1 by default for jpeg decoding ?

**Comment by chuanli1** [comment]
Yes,  this is feasible, we will put this task in the to-do list.
